test/e2e/util/util.go

```diff
@@ -914,62 +914,78 @@ func EnsureAPIUX(t *testing.T, ctx context.Context, hostClient crclient.Client,
 
 func EnsureSecretEncryptedUsingKMS(t *testing.T, ctx context.Context, hostedCluster *hyperv1.HostedCluster, guestClient crclient.Client) {
 	t.Run("EnsureSecretEncryptedUsingKMS", func(t *testing.T) {
-		// create secret in guest cluster
-		testSecret := corev1.Secret{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      "test-secret",
-				Namespace: "default",
-			},
-			Type: corev1.SecretTypeOpaque,
-			Data: map[string][]byte{
-				"myKey": []byte("myData"),
-			},
-		}
-		if err := guestClient.Create(ctx, &testSecret); err != nil {
-			t.Errorf("failed to create a secret in guest cluster; %v", err)
-		}
+		ensureSecretEncryptedUsingKMS(t, ctx, hostedCluster, guestClient, "k8s:enc:kms:")
+	})
+}
 
-		restConfig, err := GetConfig()
-		if err != nil {
-			t.Errorf("failed to get restConfig; %v", err)
-		}
+func EnsureSecretEncryptedUsingKMSV1(t *testing.T, ctx context.Context, hostedCluster *hyperv1.HostedCluster, guestClient crclient.Client) {
```

Comment:
it seems we are not using this or `EnsureSecretEncryptedUsingKMS` anywhere, can we remove them?

Reply:
I was considering keeping it around to write a test that would verify that existing KMS v1 installations are still using v1, but since I am low on time with the freeze, I was considering doing that in a separate PR.